dietpi/func/dietpi-globals

```diff
@@ -34,10 +34,18 @@
 	#	OK | systemd	= no STDIN
 	#	OK | Cron	= no STDIN
 	#	NB | /etc/profile, ~/.profile, /etc/profile.d/, /etc/bash.bashrc, ~/.bashrc and /etc/bashrc.d/ can all be interactive since those are sourced from originating shell/bash session.
-	[[ -n ${G_USER_INPUTS+x} ]] && G_INTERACTIVE=$G_USER_INPUTS
 	if [[ $G_INTERACTIVE != [01] ]]; then
 
-		[[ -t 0 ]] && G_INTERACTIVE=1 || G_INTERACTIVE=0
+		# Backwards compatibility to keep user scripts valid for a while
+		if [[ $G_USER_INPUTS == [01] ]]; then
```

Comment:
I switched to [01] testing, so in case any invalid value has been given, it is reverted to auto detection via STDIN check.